data_interrogator/interrogators.py

```diff
@@ -264,7 +265,13 @@ def get_base_annotations(self):
 
     def get_annotation(self, column):
         agg, field = column.split('::', 1)
-        if agg == 'sumif':
+        if agg == 'lookup':
+            try:
+                field, cond, value = field.split(',', 2)
```

Comment:
Will this work?

Reply:
This will work - or it will error and because in the except block with a warning.